src/amo/reducers/addonsByAuthors.js

```diff
@@ -153,7 +154,8 @@ export const getAddonsForUsernames = (
       return state.byAddonId[id];
     })
     .filter((addon) => {
-      return addonType ? addon.type === addonType : true;
+      const addonTypeFilter = getAddonTypeFilter(addonType);
+      return addonType ? addonTypeFilter.includes(addon.type) : true;
```

Comment:
What is the type of `addonTypeFilter`? It is a `string`, no? Does `includes()` work on strings?

Reply:
yep : ) [strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) and [arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) : )

Reply:
ok, cool. it is a bit black magic but I guess it is fine here.

Reply:
We should probably add a test case for this update, I see that we have a `getAddonsForUsernames selector` describe block in the test file.

Reply:
hmm i struggled with these tests. Let me know if what I have is enough :/